common/openapi.yml

```diff
@@ -249,6 +249,10 @@ components:
           type: string
           format: uuid
           description: ID of the method version to run
+        call_caching_enabled:
+          type: boolean
+          default: false
```

Comment:
Default should be `true` IMO. Yes, we could default the GUI checkbox to on and have it override the `false`, but it just seems easier to have the the whole stack swimming in the same direction.

(I think in a DM I even suggested we should not expose the ability to disable call caching, because it overshadows [superior solutions like `volatile`; but that's a topic for the GUI/another day.](https://cromwell.readthedocs.io/en/stable/optimizations/VolatileTasks/))